app/popup.js

```diff
@@ -32,16 +33,22 @@
 
   sendMessageToActiveTab({message: 'info'}, (response) => {
     if (response.message === 'inactive') {
+      getLoading().style.display = 'none';
       getStartButton().style.display = 'inline-block';
       getStopButton().style.display = 'none';
     }
 
     if (response.message === 'active') {
+      getLoading().style.display = 'none';
```

Comment:
@tabanliviu MAINTAINABILITY - Please refactor to use 'hide' so you don't need any special knownledge of layout in your JS.